src/neon/azext_neon/aaz/latest/neon/postgres/_create.py

```diff
@@ -11,10 +11,6 @@
 from azure.cli.core.aaz import *
 
 
-@register_command(
-    "neon postgres create",
-    is_preview=True,
-)
```

Comment:
why make this command unregistered

Reply:
we develop Service for 3rd party SaaS partners , as per the feedback from the team we updated the commands. please let me know if there is any concern

Reply:
If you unregistered those command, they will not be exposed as a normal commands for extension users and no one could use that command. Is it what you expected?

Reply:
yes @Kairu